app/styles/modules/_global_header.scss

```diff
@@ -0,0 +1,39 @@
+header {
+  background: #4d4d4d;
```

Comment:
Yeah agreed, I plan on extracting colors. We'll have to decide if we want to extract every color or just colors we used repeatedly.